lib/keys.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Creates keys for `private` properties on exposed objects to minimize interactions with other codebases.
+ * The key will be a Symbol if the host supports it; otherwise a prefixed string.
+ */
+var create = 
+  typeof Symbol !== 'undefined' ? 
+  function(name) {
+    return Symbol(name);
+  } 
+  :
+  function(name) {
+    return '_zone$' + name;
+  };
+
+var commonKeys = {
+  originalAddEventListener: create('originalAddEventListener'),
```

Comment:
Okay, cool. I removed the `original` prefix.

For 1), I really prefer having the common keys on a separate object. The common keys are a logical group distinct from other keys exports so IMO there's a clear benefit. I could make their export names uppercase, but that doesn't seem any more intuitive and would imply that keys create with `keys.create` elsewhere should have uppercased names for consistency. Unless there's a good reason why we shouldn't export them on their own object, I'd be happier to leave it as is.